docs/guides/cloud/introduction.mdx

```diff
@@ -4,153 +4,131 @@ sidebar_position: 10
 sidebar_label: Introduction
 ---
 
-:::tip
+[Cypress Cloud](https://on.cypress.io/cloud) is our enterprise-ready companion
+to the open-source Cypress app, allowing you to:
 
-<strong>
-  <Icon name="star" /> Get Started
-</strong>
+- [View past test results](#View-past-test-results) from your CI environment.
```

Comment:
It'd be nice to include some subtle icons here to include with each. I always think of straight text links as major points as something that could be easily missed without a visual. Especially in this "intro" sort of content. Would need to not be too marketing-y tho.

Reply:
I threw a couple in and thought it looked kinda cheesy. Do you know where else we do this?

Reply:
We don't currently do this, except for adding some brand logos. Would be more of a design task, but we might be able to repurpose things like these just to help give this some scannable definition. 

![Screen Shot 2023-06-01 at 1 07 16 PM](https://github.com/cypress-io/cypress-documentation/assets/6298501/33f0b904-7206-40f5-ba0a-7a028282b363)